versions/ubuntu_20.04_2.10.1

```diff
@@ -0,0 +1,2 @@
+2
+
```

Comment:
1. Please remove empty line.
2. What about 2.10.0?

Reply:
1. done
2. I don't think that it's need to change, because it past.

Reply:
2. We still can rebuild images of 2.10.0, for example, with new rocks versions.